libraries/architecture/src/main/kotlin/io/element/android/libraries/architecture/AsyncAction.kt

```diff
@@ -79,9 +79,9 @@ sealed interface AsyncAction<out T> {
 
     fun isUninitialized(): Boolean = this == Uninitialized
 
-    fun isConfirming(): Boolean = this is Confirming
+    fun isConfirming(): Boolean = this == Confirming
```

Comment:
Why those changes?

Reply:
Since they're both `data object`s, I thought using the equals was more expressive. It should have the same result.